addons/player-thumb/addon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Show project thumbnail under green flag overlay",
+  "description": "Shows the project's thumbnail instead of its initial state before starting the project.",
+  "credits": [
+    {
+      "name": "Samq64",
+      "link": "https://scratch.mit.edu/users/samq64/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["projects"]
+    },
+    {
+      "url": "loading.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": { "loading": true }
+      }
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"],
+      "runAtComplete": false
+    }
+  ],
+  "settings": [
+    {
+      "name": "Show while loading projects",
+      "id": "loading",
+      "type": "boolean",
+      "default": true
+    }
+  ],
```

Comment:
The description should mention this option, especially if it's enabled by default.